spec/routes/api/project/firewall_rule_spec.rb

```diff
@@ -47,7 +45,7 @@
         port_range: "80..5432"
       }.to_json
 
-      expect(last_response.status).to eq(400)
+      expect(last_response).to have_api_error(400, "cidr and port_range and firewall_id is already taken")
     end
 
     it "firewall rule no port range" do
```

Comment:
New test was added to this file in https://github.com/ubicloud/ubicloud/commit/2e24c84a1776ff6e6bbf75dc4dcd2060eb6c1ab3. Feel free to ignore and I'll update the new tests when this PR is merged.

Reply:
Thanks for the heads up. I added them.